shared/unattended/Fedora-24.ks

```diff
@@ -0,0 +1,40 @@
+install
+KVM_TEST_MEDIUM
+text
+reboot
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
```

Comment:
Interesting, I never noticed this but apparently it works fine with ppc (hvc0), arm (ttyAMA0) and s390 (ttysclp0)...